source/_integrations/acmeda.markdown

```diff
@@ -15,9 +15,17 @@ ha_platforms:
   - sensor
 ---
 
-The Rollease Acmeda Automate integration allows you to control and monitor covers via your Rolelase Acmeda Automate hub. The integration uses an [API](https://pypi.org/project/aiopulse/) to directly communicate with hubs on the local network, rather than connecting via the cloud or via RS-485.
+The Rollease Acmeda Automate integration allows you to control and monitor covers via your Rolelase Acmeda Automate hub. The integrations communicates directly with hubs on the local network, rather than connecting via the cloud or via RS-485. Devices are represented as a cover for monitoring and control as well as a sensor for monitoring battery condition.
 
-Devices are represented as a cover for monitoring and control as well as a sensor for monitoring battery condition.
+### Hub Versions
+
+Notice there are two versions of the hub referred to as v1 and v2. The v2 hub is the next generation from Rollease Acmeda. It uses the "Automate" branding, and a completely different network protocol. The v1 hub is no longer sold.
+
+The integration for the v1 hub was written by atmurray and uses uses the [aiopulse API](https://pypi.org/project/aiopulse/), also written by atmurray.
+
+The v2 hub integration was written by sillyfrog and uses the [aiopulse2 API](github.com/sillyfrog/aiopulse2) which is the aiopulse API with modifications by sillfrog. Currently the v2 Integration is awaiting review for inclusion in HA, and is designed to be added as a custom HACS repository. Instructions for installing the v2 integration can be found [here](github.com/sillyfrog/Automate-Pulse-v2).
+
+The remainder of this description applies to both hub integrations.
```

Comment:
We don't support custom components, thus should not be part of our documentation.
We generally list the supported devices instead.

```suggestion
### Supported devices

- Automate Pulse Hub v1

```